fallzahlen_kanton_total_csv/COVID19_Fallzahlen_Kanton_VS_total.csv

```diff
@@ -1,18 +1,27 @@
-2020-03-09,"",VS,"",12,"","","","","",https://www.vs.ch/de/web/coronavirus/info?p_p_id=com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN&p_p_lifecycle=0&p_p_state=normal&p_p_mode=view&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_delta=5&p_r_p_resetCur=false&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_cur=4#collapse6846567
-2020-03-10,"",VS,"",18,4,"","",3,"",https://www.vs.ch/de/web/coronavirus/info?p_p_id=com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN&p_p_lifecycle=0&p_p_state=normal&p_p_mode=view&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_delta=5&p_r_p_resetCur=false&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_cur=3#collapse6858802
-2020-03-11,"",VS,"",22,"","","","","",https://www.vs.ch/de/web/coronavirus/info?p_p_id=com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN&p_p_lifecycle=0&p_p_state=normal&p_p_mode=view&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_delta=5&p_r_p_resetCur=false&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_cur=3#collapse6870561
-2020-03-13,"",VS,"",22,"","","","",1,https://www.vs.ch/de/web/coronavirus/info?p_p_id=com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN&p_p_lifecycle=0&p_p_state=normal&p_p_mode=view&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_delta=5&p_r_p_resetCur=false&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_cur=3#collapse6898881
-2020-03-17,"",VS,"",95,"","","","",3,https://www.vs.ch/de/web/coronavirus/info?p_p_id=com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN&p_p_lifecycle=0&p_p_state=normal&p_p_mode=view&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_delta=5&p_r_p_resetCur=false&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_cur=2#collapse6927494
-2020-03-19,"",VS,"",232,"","","","",4,https://www.vs.ch/de/web/coronavirus#collapse6955818
-2020-03-20,"",VS,"",282,"","","","",5,https://www.vs.ch/de/web/coronavirus
-2020-03-21,"",VS,"",359,"","","","",9,https://www.vs.ch/de/web/coronavirus
-2020-03-22,"",VS,"",432,"","","","",10,https://www.vs.ch/de/web/coronavirus
-2020-03-23,"",VS,"",492,"","","","",12,https://www.vs.ch/de/web/coronavirus
-2020-03-24,16:00,VS,"",544,"","","","",13,https://www.vs.ch/de/web/coronavirus
-2020-03-25,15:00,VS,"",651,84,11,"","",14,https://www.vs.ch/documents/6756452/7008787/Sit%20Epid%20-%20%C3%89tat%20Stand%2025.03.2020.pdf
+2020-02-28,"",VS,"",1,3,"","","","",https://vs.ch/documents/529400/6767345/2020+02+28+-+Medienmitteilung+-+1.+Fall+Coronavirus+VS.pdf/37c4f942-c5d5-6ab5-64fd-83444de4eba5?t=1582923242156
```

Comment:
Weird that they report 1 first positive case and then 3 hospitalized for the same date :) ? Probably hospitalized before there was confirmation?

Reply:
I assumend the same. Should we add a column "remark" to have a place the add these information if confirmed? Sooner or later we have to double check these kind of information to improve the data quality.